src/_reducers/assets.reducer.js

```diff
@@ -17,7 +17,12 @@ export default (state = initialState.assets, action) => {
       return {
         ...state,
         hasError: false,
-        isLoading: true
+        isLoading: true,
+        messages: {
```

Comment:
Having the status message in an object is over-engineering. Look at the comment in the `initialState` file to understand what I'm talking about